test/test_model.py

```diff
@@ -72,13 +69,16 @@ def config_get(self):
             'qux': True,
         }
 
-
 class TestModel(unittest.TestCase):
 
     def setUp(self):
         self.backend = FakeModelBackend()
         self.model = op.model.Model('myapp/0', ['db0', 'db1', 'db2'], self.backend)
 
+        os.environ['JUJU_UNIT_NAME'] = 'myapp/0'
+
+        self.addCleanup(os.environ.pop, 'JUJU_UNIT_NAME')
```

Comment:
Empty line might go away. Line below is strictly related to the one above.

Sorry for the nitpick. I'm hoping to help establishing that kind of common project convention so we don't need to discuss this in the future.